sdk/core/Azure.Core/tests/TestFramework/ConnectionString.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Core.Testing
+{
+    public class ConnectionString
```

Comment:
Is this the third ConnectionString implementation we've written?  We should really put one in Azure.Core.

Reply:
Didn't want to open that can of worms as part of this PR. But we should.